core/templates/dev/head/domain/question/QuestionBackendApiServiceSpec.ts

```diff
@@ -103,109 +103,63 @@ describe('Question backend Api service', function() {
     var successHandler = jasmine.createSpy('success');
     var failHandler = jasmine.createSpy('fail');
 
-    $httpBackend.expect(
-      'GET', '/question_player_handler?skill_ids=1&question_count=1').respond(
+    var questionPlayerHandlerUrl =
+      '/question_player_handler?skill_ids=1&question_count=1' +
+      '&fetch_by_mastery=true';
+    $httpBackend.expect('GET', questionPlayerHandlerUrl).respond(
       sampleDataResults);
     QuestionBackendApiService.fetchQuestions(
-      ['1'], 1).then(successHandler, failHandler);
+      ['1'], 1, true).then(successHandler, failHandler);
     $httpBackend.flush();
 
     expect(successHandler).toHaveBeenCalledWith(
       sampleDataResults.question_dicts);
     expect(failHandler).not.toHaveBeenCalled();
   });
 
-  it('should successfully fetch questions from the backend with given cursor',
```

Comment:
Why are so many tests deleted?

Reply:
I think only the cursor tests were deleted. That makes sense because there's no concept of a cursor anymore for the question player

Reply:
Yes, only the cursor tests were deleted. This concept was removed before, but these tests weren't removed by accident.

Reply:
But then why did the non-removed tests still pass when the concept was removed?

Reply:
The cursor parameter was removed from the testing URL, so the test is basically the same as the other tests without cursors.